modules/nf-core/shinyngs/app/tests/main.nf.test

```diff
@@ -0,0 +1,103 @@
+nextflow_process {
+
+    name "Test Process SHINYNGS_APP"
+    script "../main.nf"
+    process "SHINYNGS_APP"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "shinyngs"
+    tag "shinyngs/app"
+
+    test("multi - tsv") {
```

Comment:
From [the migration guide to nf-test](https://nf-co.re/docs/contributing/modules#migrating-from-pytest-to-nf-test): provide a test name preferably indicating the test-data and file-format used. Example: test("homo_sapiens - [bam, bai, bed] - fasta - fai")